src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+const li = document.querySelectorAll('li');
```

Comment:
Consider renaming the variable 'li' to something more descriptive like 'employeeList' for better readability.